modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceAbstractIntegrationTest.java

```diff
@@ -352,7 +424,7 @@ public HttpResponse sendLoginPost(HttpClient client, String sessionDataKey)
 	 * @param password       Password.
 	 * @return Http response.
 	 * @throws ClientProtocolException
-	 * @throws IOException
+	 * @throws  IOException
```

Comment:
Why this line is changed?

Reply:
Fixed with https://github.com/wso2/product-is/pull/15906/commits/38e6cdd0e3f7357405e8fe1891714dcfcb17624b